src/scripting_api/doc.js

```diff
@@ -108,12 +111,22 @@ class Doc extends PDFObject {
         "DidPrint",
         "OpenAction",
       ]);
+      this._disableSaving = true;
```

Comment:
Given that saving is listed in the `dontRun` Set above, why do we need to explicitly disable saving here?

---

If this actually is necessary, a comment would probably be a good idea.
Furthermore, if this is intended, why don't we set `disablePrinting` here as well?

Reply:
> If this actually is necessary, a comment would probably be a good idea. Furthermore, if this is intended, why don't we set `disablePrinting` here as well?

Because it'll disable auto-print.